src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -206,6 +206,10 @@ void DlgPrefLibrary::slotResetToDefaults() {
     radioButton_dbclick_bottom->setChecked(false);
     radioButton_dbclick_top->setChecked(false);
     radioButton_dbclick_deck->setChecked(true);
+    radioButton_cover_art_fetcher_highest->setChecked(false);
+    radioButton_cover_art_fetcher_high->setChecked(false);
+    radioButton_cover_art_fetcher_medium->setChecked(false);
+    radioButton_cover_art_fetcher_lowest->setChecked(true);
```

Comment:
Since these radio buttons are in the same widget "groupBox_cover_art_fetcher" you only need to check one to disable the others. In slotUpdate() it's done correctly.
This also applies to the other radio buttons (Double-click action) where you copied this behavior from.

> Radio buttons are autoExclusive by default. If auto-exclusive is enabled, radio buttons that belong to the same parent widget behave as if they were part of the same exclusive button group. 
https://doc.qt.io/qt-6/qradiobutton.html#details

Feel free to pick 3f66b569a3